scripts/minikube.sh

```diff
@@ -162,11 +162,21 @@ else
 fi
 
 #configure csi sidecar version
-CSI_ATTACHER_VERSION=${CSI_ATTACHER_VERSION:-"v3.2.1"}
-CSI_SNAPSHOTTER_VERSION=${CSI_SNAPSHOTTER_VERSION:-"v4.1.1"}
-CSI_PROVISIONER_VERSION=${CSI_PROVISIONER_VERSION:-"v2.2.2"}
-CSI_RESIZER_VERSION=${CSI_RESIZER_VERSION:-"v1.2.0"}
-CSI_NODE_DRIVER_REGISTRAR_VERSION=${CSI_NODE_DRIVER_REGISTRAR_VERSION:-"v2.2.0"}
+
+# as we are no longer keeping the sidecar versions here, try to fetch it from build.env file
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+source "${SCRIPT_DIR}/../build.env"
+
+sidecars=(CSI_ATTACHER_VERSION CSI_SNAPSHOTTER_VERSION CSI_PROVISIONER_VERSION CSI_RESIZER_VERSION CSI_NODE_DRIVER_REGISTRAR_VERSION)
+for controller in "${sidecars[@]}"; do
+	if [[ "${!controller}" ]]; then
```

Comment:
For checking empty strings, use `-z`.
No need for listing the versions, that is just clutter IMHO.
This check is not always needed, only for deploying?

Reply:
sure, used `-z`